roadside-forms-frontend/frontend_web_app/src/components/Forms/TwentyFourHourForm/twentyFourHourForm.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { VehicleImpoundment } from './component/vehicleImpoundment';
+import { Prohibition } from './component/prohibition';
+import { ReasonableGrounds } from './component/reasonableGrounds';
+import PropTypes from 'prop-types';
+
+export const TwentyFourHourForm = (props) => {
+    const {cities, impoundLotOperators} = props;
+ console.log(impoundLotOperators);
+  return (
+    <div>
+    <VehicleImpoundment impoundLotOperators={impoundLotOperators}></VehicleImpoundment>
+    <Prohibition cities={cities}></Prohibition>
+    <ReasonableGrounds></ReasonableGrounds>
```

Comment:
Same here all of these components can be done with one tag 
<VehicleImpoundment impoundLotOperators={} />

Reply:
when you say one tag, do you mean <tag/> instead of opening and closing tag?